content/en/docs/reference/kubectl/overview.md

```diff
@@ -255,17 +255,17 @@ $ kubectl get pods --sort-by=.metadata.name
 
 Use the following set of examples to help you familiarize yourself with running the commonly used `kubectl` operations:
 
-`kubectl create` - Create a resource from a file or stdin.
+`kubectl apply` - Apply or Update a resource from a file or stdin.
 
```

Comment:
@Liujingfang1, does `apply` include create and update [instead of apply and update]?

Reply:
Bases on status of objects on the cluster, `apply` takes different actions. If the applied object exists on the cluster, apply will do an update to it. If the applied object doesn't exist on the cluster, apply will create it.